invokeai/frontend/web/tsconfig.json

```diff
@@ -13,6 +13,8 @@
     "moduleResolution": "Node",
     // TODO: Disabled for IDE performance issues with our translation JSON
     // "resolveJsonModule": true,
+    "noUncheckedIndexedAccess": true,
+    "strictNullChecks": true,
```

Comment:
Tightened up the reins a bit. Ran into some bugs related to indexed array access: TS does not type indexed array access values as a union with `undefined`, so it's very easy to end up with undefined when working with arrays.

Enabling this rule unions *all* indexed array access values with `undefined`. Many more runtime checks are needed, but no more errors.